webapp/controller/Home.controller.js

```diff
@@ -1,19 +1,136 @@
-sap.ui.define(['ui5boilerplate/controller/Base.controller'], (BaseController) => {
-  return BaseController.extend('ui5boilerplate.controller.Home', {
-    onInit() {},
+/* eslint-disable object-shorthand */
+//
+sap.ui.define(
+  ['sap/ui/core/mvc/Controller', 'sap/ui/model/json/JSONModel', 'sap/m/MessageToast'],
+  (Controller, JSONModel, MessageToast) => {
+    return Controller.extend('ui5boilerplate.controller.Home', {
+      /*  OnInit - lifecycle function
+       */
+      onInit() {
+        const membership = { membership: [] };
+        /* Create JSON Model
+         */
+        var OModel = new JSONModel();
+        var OModel1 = new JSONModel();
+        /* Load JSON file data in synchronous mode
+         */
+        OModel.loadData('/model/bachelor_courses.json', false);
+        /* named model binding, useful when we have multiple model in the same view
+         */
+        this.getView().setModel(OModel, 'courseModel');
+        /* JSON model for membership
+         */
+        OModel1.setData(membership);
+        this.getView().setModel(OModel1, 'membership');
+        this.setMaxDobDate();
+      },
 
-    /**
-     * Press event handler for INTEGRTR logo
-     */
-    integrtrLogoPress() {
-      window.open('https://www.integrtr.com', '_blank');
-    },
+      /**
+       * Set maxDate for DatePicker control
+       * @function
+       */
+      setMaxDobDate() {
+        const dob_picker = this.getView().byId('dob_picker');
```

Comment:
Can be handled using property binding as well. That would be better in terms of scalability and maintenance.

Ex: maxDate="{model>/maxDateAllowed}"